modules/ppcp-settings/src/Endpoint/LoginLinkRestEndpoint.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/**
+ * REST endpoint to manage the onboarding module.
+ *
+ * @package WooCommerce\PayPalCommerce\Settings\Endpoint
+ */
+
+declare( strict_types = 1 );
+
+namespace WooCommerce\PayPalCommerce\Settings\Endpoint;
+
+use WP_REST_Server;
+use WP_REST_Response;
+use WP_REST_Request;
+use WooCommerce\PayPalCommerce\Settings\Service\ConnectionUrlGenerator;
+
+/**
+ * REST controller that generates merchant login URLs.
+ */
+class LoginLinkRestEndpoint extends RestEndpoint {
+	/**
+	 * The base path for this REST controller.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'login_link';
```

Comment:
```suggestion
	protected string $rest_base = 'login_link';
```
Same here, is there a reason to not use the type-hints for this specific property?